src/Facebook/InstantArticles/Elements/Div.php

```diff
@@ -32,14 +32,10 @@ public static function create()
      *
      * @param \DOMDocument $document - The document where this element will be appended (optional).
      *
-     * @return \DOMElement
```

Comment:
Are `DOMElement` and `DOMNode` the same?

Reply:
Technically all DOMElement is a DOMNode, but not the other way around.

While parsing a DOMDocument and navigating through the children elements, only standard tags will map to DOMElement.